doc/source/whatsnew/v0.18.2.txt

```diff
@@ -38,6 +38,8 @@ Other enhancements
      idx = pd.Index(["a1a2", "b1", "c1"])
      idx.str.extractall("[ab](?P<digit>\d)")
 
+- Support decimal option in PythonParser (:issue:`12933`)
```

Comment:
`
so something like

``pd.read_csv()`` with ``engine='python'`` gained support for the ``decimal`` option.